### PR TITLE
valve_analitics_webapp: add project files

### PR DESCRIPTION
**Pull request by PashaDem**

### CODE REVIEW

File: valve_analytics_webapp/settings.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-nxijwdxi-cfzdg@yx1a)otit&jrgx8us-ni+hla@ks91*mrblt"
```
Comment (mosey20000) [inline valve_analytics_webapp/settings.py:23]: hide secret key to .env and create .env.tmp

Comment (PashaDem) [reply valve_analytics_webapp/settings.py]: Updated

**Review by Vladgasik — CHANGES_REQUESTED** [review]
Remove auto-generated comments in `settings.py`

**Comment by PashaDem** [comment]
> Remove auto-generated comments in `settings.py`

Updated

**Review by Vladgasik — DISMISSED** [review]

File: valve_analytics_webapp/settings.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+
+INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+    "rest_framework",
```
Comment (Vladgasik) [inline valve_analytics_webapp/settings.py:22]: Use this approach, please

image

Comment (PashaDem) [reply valve_analytics_webapp/settings.py]: Updated

**Review by Vladgasik — APPROVED** [review]

**Pull request merged**